agent/consul/client_test.go

```diff
@@ -78,7 +79,9 @@ func testClientWithConfigWithErr(t *testing.T, cb func(c *Config)) (string, *Cli
 		t.Fatalf("err: %v", err)
 	}
 
-	client, err := NewClient(config, WithLogger(logger), WithTLSConfigurator(tlsConf))
+	store := &token.Store{}
+
+	client, err := NewClient(config, WithLogger(logger), WithTLSConfigurator(tlsConf), WithTokenStore(store))
```

Comment:
Apparently we were running all test clients without a token store in 1.8.x and the recent additions caused plenty of test failures now that we expect to be able to retrieve a token from it.